app/views/admin/cycles/plugin_relations/petitions/index.html.slim

```diff
@@ -39,8 +39,11 @@
          h3 Texto completo 
        .filters.clearfix
          .pull-right
-           = link_to 'Baixar documento', @petition.current_version.document_url, class: 'btn', target: '_blank' if @petition.current_version.document_url
-       - if @petition.current_version.body
+           - if @petition.published_version
+             = link_to 'Baixar documento', @petition.published_version.document_url, class: 'btn', target: '_blank'
+           - else
+             = 'Nenhuma versão publicada ainda'
+       - if @petition.current_version
```

Comment:
Here I don't think we need to use the published version, if we use things will look strange to the user, because after he edits de petition he is redirected to this page, and since the publishing process is asynchronous the content will be empty, I think that will look weird to the user, since he has just edited content.